tests/nat/front_ends/fastapi/test_dask_client_mixin.py

```
@@ -0,0 +1,136 @@
+            mock_client_class.assert_called_once_with(address=test_address)
+            assert client is mock_blocking_client
+
+
+def test_blocking_client_exception_handling(dask_client_mixin, mock_blocking_client):
+    """Test blocking client is closed even when an exception occurs."""
+    test_address = "tcp://127.0.0.1:8786"
+
+    with patch('nat.front_ends.fastapi.dask_client_mixin.Client', return_value=mock_blocking_client):
+        with pytest.raises(RuntimeError, match="Test exception"):
+            with dask_client_mixin.blocking_client(test_address):
+                raise RuntimeError("Test exception")
```

Ruff (0.13.0)

Avoid specifying long messages outside the exception class

(TRY003)